bigbluebutton-html5/imports/ui/components/actions-bar/emoji-menu/component.jsx

```diff
@@ -0,0 +1,179 @@
+import React, { Component, PropTypes } from 'react';
+import { defineMessages, injectIntl } from 'react-intl';
+
+import Button from '/imports/ui/components/button/component';
+import Dropdown from '/imports/ui/components/dropdown/component';
+import DropdownTrigger from '/imports/ui/components/dropdown/trigger/component';
+import DropdownContent from '/imports/ui/components/dropdown/content/component';
+import DropdownList from '/imports/ui/components/dropdown/list/component';
+import DropdownListItem from '/imports/ui/components/dropdown/list/item/component';
+import DropdownListSeparator from '/imports/ui/components/dropdown/list/separator/component';
+
+const propTypes = {
+  // Emoji status of the current user
+  userEmojiStatus: PropTypes.string.isRequired,
+  actions: PropTypes.object.isRequired,
+};
+
+class EmojiMenu extends Component {
+  constructor(props) {
+    super(props);
+  }
+
+  render() {
+    const {
+     userEmojiStatus,
+     actions,
+     intl,
+   } = this.props;
+
+    return (
+      <Dropdown ref="dropdown">
+        <DropdownTrigger>
+          <Button
+            role="button"
+            label={intl.formatMessage(intlMessages.statusTriggerLabel)}
+            icon="hand"
+            ghost={false}
+            circle={true}
+            hideLabel={false}
+            color="primary"
+            size="lg"
+
+            // FIXME: Without onClick react proptypes keep warning
+            // even after the DropdownTrigger inject an onClick handler
+            onClick={() => null}
+          />
+        </DropdownTrigger>
+        <DropdownContent placement="top left">
+          <DropdownList>
+            <DropdownListItem
+              icon="time"
```

Comment:
this emoji should match the string "away". Please check the rest of the status strings in my other comment